sample/F8/sessions

```diff
@@ -0,0 +1,1470 @@
+[
+   {
+      "short_abstract":"",
+      "id":5,
+      "subtitle":"introduction-to-the-camera-effects-platform",
+      "title":"Introduction to the Camera Effects Platform",
+      "long_abstract":"Come hear more about Camera Effects Platform, a suite of creative tools that lets you create effects for the Facebook camera, from simple photo frames to interactive masks and effects that use Augmented Reality (AR). In this session you’ll learn what camera effects are, how to create and submit your own, and how you can connect with the audiences that matter to you by providing rich experiences that help them express themselves.",
+      "comments":"",
+      "language":"",
+      "slides":"",
+      "audio":"",
+      "video":"",
+      "signup_url":"",
+      "state":"accepted",
+      "session_type":{
+         "id":6,
+         "name":"50 minutes session",
+         "length":"00.50"
+      },
+      "track":{
+         "id":2,
+         "name":"CAMERA",
+         "color":"#FF3046"
+      },
+      "end_time":"2017-04-18T13:20:00",
+      "start_time":"2017-04-18T12:30:00",
+      "speakers":[
+         {
+            "city":"",
+            "heard_from":"",
+            "icon":"",
+            "long_biography":"",
+            "small":"",
+            "photo":"",
+            "thumbnail":"",
```

Comment:
add speaker images. theres no point of an event without the images. also provide relevant info of speaekers.